appveyor.yml

```diff
@@ -61,6 +61,16 @@ environment:
       PYTHON_ARCH: "64"
       UIA_SUPPORT: "NO"
 
+    - PYTHON: "C:\\Python36"
+      PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "32"
+      UIA_SUPPORT: "NO"
+
+    - PYTHON: "C:\\Python36-x64"
+      PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "64"
+      UIA_SUPPORT: "YES"
+
```

Comment:
Not sure it's time to drop Py3.3 support. Maybe later. And... more pythons - more chances to reproduce unstable test cases that we fight with.